go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/docker/go-units v0.4.0 // indirect
 	github.com/dustinkirkland/golang-petname v0.0.0-20170921220637-d3c2ba80e75e
+	github.com/go-logr/logr v0.1.0
```

Comment:
another logger?

Reply:
haha sharp eye, I wondered if someone catches that :) but no, this is just an interface of the logger we were using before :) but I now had to explicitly referenced the type so we have the explicit dependency